src/libs/brands.js

```diff
@@ -187,7 +187,8 @@ export const brands = {
     logos: {
       color: radXLogo,
       white: radXLogoWhite
-    }
+    },
+    catalogConsortiumsToInclude: ['RADx-UP', 'RADx-DHT', 'RADx-Rad', 'RADx-Tech']
```

Comment:
@kyuksel this you specifically had thoughts on

Reply:
I think you mentioned this in the meeting... should "consortiums" be "consortia"?